packages/sources/uniswap-v2/README.md

```diff
@@ -0,0 +1,71 @@
+# Chainlink External Adapter for UniswapV2.fi
+
+This adapter allows querying UniswapV2.fi contracts
+
+### Environment Variables
+
+| Required? |        Name        |                                                Description                                                | Options |                 Defaults to                  |
+| :-------: | :----------------: | :-------------------------------------------------------------------------------------------------------: | :-----: | :------------------------------------------: |
+|    ✅     |      RPC_URL       |               An http(s) RPC URL to a blockchain node that can read the UniswapV2 contracts               |         |                                              |
+|           | BLOCKCHAIN_NETWORK |    The uniswap router address to get price from. NOTE: THIS SHOULD NOT BE CHANGED ON ETHEREUM MAINNET     |         | `0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D` |
+|           |  ROUTER_CONTRACT   | The network to get pre-defined token addresses from. NOTE: THIS SHOULD NOT BE CHANGED ON ETHEREUM MAINNET |         |                  `ethereum`                  |
```

Comment:
Looks like the names of these two are switched

Reply:
Nvm fixed in f054d38d92a2e5aefa10e6ddaa45f79f39d75a27 :sweat_smile:

Reply:
woops, yea I just noticed that and pushed a typo commit